src/recur.cpp

```diff
@@ -90,7 +90,6 @@ void handleRecurrence ()
 
           Task rec (t);                          // Clone the parent.
           rec.setStatus (Task::pending);         // Change the status.
-          rec.id = Context::getContext ().tdb2.next_id ();      // New ID.
```

Comment:
No, the id's are set based on the task's position in the working set, which comes from TaskChampion. Next time this task is read from the DB it will have its correct ID.